sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/download/manager/DownloadManagerReleaseDownloader.java

```diff
@@ -249,6 +250,9 @@ synchronized void onDownloadError(RuntimeException e) {
     }
 
     private boolean isDownloadedFileValid() {
+        if (mReleaseDetails.getFileExtension() != FileExtension.apk) {
```

Comment:
Let's leave a comment here, because looking at code it is not clear what happens. "If the file is not apk, then it is valid" doesn't make sense, so we need to clarify why this condition exists here.